.github/workflows/c-zephyr-tests.yml

```diff
@@ -39,10 +39,21 @@ jobs:
           path: core/src/main/resources/lib/c/reactor-c
           ref: ${{ inputs.runtime-ref }}
         if: ${{ inputs.runtime-ref }}
-      - name: Perform Zephyr tests for C target with default scheduler
+      - name: Run Zephyr smoke tests
         run: |
-          ./gradlew targetTest -Ptarget=CZephyr
+          ./gradlew core:integrationTest --tests org.lflang.tests.runtime.CZephyrTest.buildZephyr*
```

Comment:
What is the reason for factoring this out into smaller steps?

Reply:
The thought was to run each test category to completion before starting the next. I had some cases where it run for 40m before returning the error which could have been caught and reported in 5m if we did each test-category to completion.